docs/01_manual.md

```diff
@@ -81,81 +81,101 @@ For details about the service API see the [OpenAPI specification].
 ARGUMENTS
 ---------
 
-      -b, --base-dir  <arg>     BagStore base-dir to use
+      -b, --base-dir  <arg>     bag store base-dir to use
       -s, --store-name  <arg>   Configured store to use
           --help                Show help message
           --version             Show version of this program
     
-    Subcommand: list - Lists the bag-stores for which a shortname has been defined. 
-    These are the bag-stores that are accessible through the HTTP interface
+    Subcommand: list - Lists the bag stores for which a shortname has been defined. These are the bag stores
+    that are also accessible through the HTTP interface.
     
           --help   Show help message
+    ---
+    
     Subcommand: add - Adds a bag to the bag-store
-      -u, --uuid  <arg>   UUID to use as name for the Bag
+      -m, --move          move (rather than copy) the bag when adding it to the bag
+                          store
+      -u, --uuid  <arg>   UUID to use as bag-id for the bag
           --help          Show help message
     
      trailing arguments:
-      bag (required)   the (unserialized) Bag to add
+      bag (required)   the (unserialized) bag to add
     ---
     
-    Subcommand: get - Retrieves a Bag or File in it
-          --help   Show help message
+    Subcommand: get - Retrieves an item by copying it to the specified directory (default: current directory).
```

Comment:
Changed the interface of `get` so that it will copy to the current directory by default. Also added a `stream` subcommand that will copy to stdout in TAR or ZIP packaging format.